lti_consumer/lti_xblock.py

```diff
@@ -248,8 +248,21 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
         default="",
         scope=Scope.settings
     )
+    config_type = String(
+        display_name=_("Configuration Type"),
+        scope=Scope.settings,
+        values=[
+            {"display_name": _("New Configuration"), "value": "new"},
+            {"display_name": _("Existing Configuration"), "value": "external"},
```

Comment:
The wording here needs to be improved, this can cause some confusion.

Reply:
I think this calls for product help. @marcotuts Is the wording here something you can help us with?

- New configuration: It's actually when authors configure LTI directly on the block (maybe something like `On block`?).
- Existing Configuration: External configuration, where edX staff/admins provide an LTI config string, which is reusable across the courses.

Reply:
@marcotuts Can you kindly take a look at this when you have some time? Your inputs would be really helpful in moving this PR forward.

Reply:
@tecoholic Lets change this to:

- `Configuration on block`: default value - config on Xblock.
- `Reusable configuration`: new setting we are introducing with this PR.